plugins/rikki/loungeviews/components/Profile.php

```diff
@@ -20,8 +20,8 @@ public function componentDetails()
     public $profile = null;
     public $sloth = null;
     public $user = null;
-    public $battleTagFormmattted = null;
-    public $hp_region = null;
+    public $battleTagFormatted = null;
+    public $region = null;
```

Comment:
$region is not an ideal name since we already have a region attached here (and it is exactly NOT the one that is variable holds) - maybe heroesprofile_region_id